projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/input-docs/input-docs.component.html

```diff
@@ -351,4 +351,52 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
 
     </div>
   </go-card>
+
+  <go-card id="form-length-type" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2 go-heading--no-wrap">Component Maxlength And Minlength</h2>
+      <go-copy cardId="form-length-type" goCopyCardLink></go-copy>
```

Comment:
Should the `cardId` and and the `id` above be `form-input-max-min-length` or something similar to follow the pattern of the other cards?